website/docs/reference/resource-properties/unit-tests.md

```diff
@@ -155,7 +155,60 @@ When `format: csv`, can either supply:
 
 ### Overrides
 
-When configuring your unit test, you can override the output of macros, vars, or environment variables for a given unit test. 
+When configuring your unit test, you can override the output of [macros](https://docs.getdbt.com/docs/build/jinja-macros#macros), [project variables](https://docs.getdbt.com/docs/build/project-variables), or [environment variables](https://docs.getdbt.com/docs/build/environment-variables) for a given unit test. 
+
+```yml
+
+unit_tests:
+  - name: my_favorite_unit_test
+    model: my_favorite_model
+    overrides:
+      macros:
+        is_incremental: false 
+        current_timestamp: "date('2023-01-15')"
```

Comment:
We'll want to replace this will a cross-database example when we can

Reply:
agree! i don't think that's a blocker for merging this in though